src/components/AssetSelector.tsx

```diff
@@ -11,30 +11,39 @@ const logger = Logger('AssetSelector');
 export interface AssetSelectorProps {
   network: NetworkInfo | undefined;
   payment: Payment;
+  withQuote: boolean;
   asset: CryptoAsset | undefined;
   onChange: (asset?: CryptoAsset) => void;
 }
 
 export const AssetSelector = ({
```

Comment:
What is the difference between a `AssetSelector` and `CurrencySelector`?

Reply:
AssetSelector is dedicated to tokens selection (for payment), and CurrencySelector is for selection of the base payment currency. Each currency has its subset of Assets (tokens).